binaries.json

```diff
@@ -0,0 +1,38 @@
+{
+    "released": [
```

Comment:
New release just happened, so these lists are identical now. Also, please move it under the `docker/tests` dir for now.

Reply:
Seems to be resolved now